southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImpl.java

```diff
@@ -44,6 +48,14 @@
     private static final Logger LOG = LoggerFactory.getLogger(AnsibleCommandServiceImpl.class);
     private Map<Uuid, ManagedProcess> processMap = new ConcurrentHashMap<>();
     private final DataBroker dataBroker;
+    private static final String ansibleVar = "ansibleVar";
+    private static final String roleVar = "roleVar";
+    private static final Map<String, String> VAR_MAP = new HashMap<String, String>() {
```

Comment:
leave in camel notation, varMap

Reply:
camel, or all caps VARMAP to make all constants uppercase?

Reply:
VAR_MAP isn''t a constant its a variable so it it typical camel

Reply:
it is a constant right? its static and final